src/js/drop-down-menu.jsx

```diff
@@ -48,12 +48,14 @@ var DropDownMenu = React.createClass({
       'mui-open': this.state.open
     });
 
+    var selectedIndex = this.state.selectedIndex > -1 ? this.state.selectedIndex : 0;
```

Comment:
I would rather right

var selectedIndex = this.state.selectedIndex;
selectedIndex = (selectedIndex > -1)  ? selectedIndex : 0;

for a better optimisation with uglifyjs